modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -0,0 +1,323 @@
+ * permit persons to whom the Software is furnished to do so, subject to the following   *
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#include <modules/fieldlinessequence/rendering/renderablefieldlinessequence.h>
+
+#include <openspace/engine/openspaceengine.h>
+#include <openspace/rendering/renderengine.h>
+#include <openspace/util/updatestructures.h>
+
+#include <ghoul/opengl/programobject.h>
+#include <ghoul/opengl/textureunit.h>
+
+
+namespace {
+    std::string _loggerCat = "RenderableFieldlinesSequence";
+
+    const GLuint _VA_POSITION = 0; // MUST CORRESPOND TO THE SHADER PROGRAM
+    const GLuint _VA_COLOR    = 1; // MUST CORRESPOND TO THE SHADER PROGRAM
+    const GLuint _VA_MASKING  = 2; // MUST CORRESPOND TO THE SHADER PROGRAM
+} // namespace
+
+namespace openspace {
+
+void RenderableFieldlinesSequence::deinitialize() {
+    glDeleteVertexArrays(1, &_vertexArrayObject);
+    _vertexArrayObject = 0;
+
+    glDeleteBuffers(1, &_vertexPositionBuffer);
+    _vertexPositionBuffer = 0;
+
+    glDeleteBuffers(1, &_vertexColorBuffer);
+    _vertexColorBuffer = 0;
+
+    glDeleteBuffers(1, &_vertexMaskingBuffer);
+    _vertexMaskingBuffer = 0;
+
+    RenderEngine& renderEngine = OsEng.renderEngine();
+    if (_shaderProgram) {
+        renderEngine.removeRenderProgram(_shaderProgram);
+        _shaderProgram = nullptr;
+    }
+
+    // Stall main thread until thread that's loading states is done!
+    while (_isLoadingStateFromDisk) {
+        LWARNING("TRYING TO DESTROY CLASS WHEN A THREAD USING IT IS STILL ACTIVE");
+    }
+}
+
+bool RenderableFieldlinesSequence::isReady() const {
+    return _isReady;
+}
+
+void RenderableFieldlinesSequence::render(const RenderData& data, RendererTasks&) {
+    if (_activeTriggerTimeIndex != -1) {
+        _shaderProgram->activate();
+
+        // Calculate Model View MatrixProjection
+        const glm::dmat4 ROT_MAT = glm::dmat4(data.modelTransform.rotation);
```

Comment:
Regular camelCase for constants. ALL_CAPS indicates preprocessor macros.

Reply:
Aaah, alright! Will fix it!